src/components/AdminPortal/AssignmentTypeSelector/AssignmentTypeSelector.tsx

```
@@ -1,5 +1,7 @@
 import React, { useId } from 'react';
-import { Chip, Box, Typography } from '@mui/material';
+import Chip from '@mui/material/Chip';
```

ESLint

[error] '@mui/material/Chip' import is restricted from being used. Do not import Chip from `@mui/material`. Use the shared StatusBadge component instead.

(no-restricted-imports)